### PR TITLE
HCAP - 1245 Fixed callback issue with edit participant component

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
It looks like we were trying to call an undefined submissionCallback function. 
I added a check to ensure it exists. I also added a fetch data call to the onClose function for the EditParticipant component in the ParticipantDetailsView to load the updated data after it's submitted.

### CODE REVIEW

**Review by fwkendall — APPROVED** [review]

File: client/src/pages/private/ParticipantDetailsView.js [hunk 1]
```diff
@@ -304,7 +304,17 @@ export default () => {
               initialValues={actualParticipant}
               validationSchema={EditParticipantFormSchema}
               onSubmit={onUpdateInfo}
-              onClose={() => setShowEditModal(false)}
+              onClose={() => {
+                setShowEditModal(false);
+                fetchData({
```
Comment (fwpushan) [inline client/src/pages/private/ParticipantDetailsView.js:309]: 1. Is this section updating the Participant table?

**Review by fwpushan — COMMENTED** [review]
LGTM, please check my comments.

**Review by fwpushan — APPROVED** [review]

File: client/src/pages/private/ParticipantDetailsView.js [hunk 2]
```diff
@@ -304,7 +304,17 @@ export default () => {
               initialValues={actualParticipant}
               validationSchema={EditParticipantFormSchema}
               onSubmit={onUpdateInfo}
-              onClose={() => setShowEditModal(false)}
+              onClose={() => {
```
Comment (arranfw) [inline client/src/pages/private/ParticipantDetailsView.js:307]: Looks like I missed an instance of `EditParticipantForm` 🤔 heads up that `validationSchema` and `onSubmit` aren't used by the component anymore, if this page uses the same component but has a different edit function `EditParticipantForm` might need to be looked at again.

Do you know if it's intentional that these two pages have different onSubmits?

**Comment by dbayly-freshworks** [comment]
I am going to merge this, and then put up a separate PR with the related test.

**Pull request merged**